client/my-sites/post-type-list/index.jsx

```diff
@@ -26,7 +26,7 @@ import PostTypeListEmptyContent from './empty-content';
 /**
  * Constants
  */
-const DEFAULT_POST_ROW_HEIGHT = 86;
+const DEFAULT_POST_ROW_HEIGHT = 81;
```

Comment:
Was this a bit too tall before?

Reply:
The `PostItem` got shorter with the removal of the clock icon in this PR.  Also related to the author weirdness below, this was probably in line previously.